src/test/java/com/wikia/webdriver/common/dataprovider/MercurySEODataProvider.java

```diff
@@ -0,0 +1,90 @@
+            }, {
+                new Page("sktest123", "/wiki/Special:Version"), "Version - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Special:Videos"), "Videos on this wiki - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Special:NewFiles"), "Images - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Blog:Recent_posts"), "Blog:Recent posts - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/User_blog:Sktest/test_blog_1"), "User blog:Sktest/test blog 1 - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Special:Forum"), "Forum - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Board:General_Discussion"), "General Discussion board - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Thread:2610"), "Test post - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Category:Premium_Videos"), "Category:Premium Videos - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Category:Non-premium_Videos"), "Category:Non-premium Videos - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Category:Premium"), "PremiumVideos - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Message_Wall:Sktest"), "Message Wall:Sktest - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Category:Thread:2160"), "Welcome to Sktest123 Wiki! - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Special:Maps"), "Maps - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Template:Welcome"), "Template:Welcome - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/MediaWiki:Common.css"), "MediaWiki:Common.css - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/MediaWiki:Edit"), "MediaWiki:Edit - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/User:Sktest"), "User:Sktest - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/File:THE_HOBBIT_Trailer_-_2012_Movie_-_Official_HD"), "Video - THE HOBBIT Trailer - 2012 Movie - Official HD - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/File:Giant_prominence_on_the_sun_erupted.jpg"), "Image - Giant prominence on the sun erupted.jpg - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Special:NonExistingSpecialPage"), "Error - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("es.pokemon", "/wiki/WikiDex"), "WikiDex - Wikia"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "es.pokemon" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)